### PR TITLE
BestNotes.BestNotes version 1.3.2

### PR DESCRIPTION
**Pull request by pikcube**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/90786)

### CODE REVIEW

**Comment by pikcube** [comment]
This is the link on the https://www.bestnotes.com/downloads/ page, it links to storage in google. Note that in the page inside google, it references BestNotes's website

File: manifests/b/BestNotes/BestNotes/1.3.2/BestNotes.BestNotes.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: BestNotes.BestNotes
+PackageVersion: 1.3.2
+PackageLocale: en-US
+Publisher: BestNotes
```
Comment (mdanish-kh) [inline manifests/b/BestNotes/BestNotes/1.3.2/BestNotes.BestNotes.locale.en-US.yaml:7]:
```suggestion
Publisher: BestNotes, LLC
PublisherUrl: https://www.bestnotes.com/
PublisherSupportUrl: https://www.bestnotes.com/support-training/
PrivacyUrl: https://www.bestnotes.com/privacy-policy/
```
Publisher field must match Control Panel / Apps&Features. Suggested some additional metadata as well.
image

File: manifests/b/BestNotes/BestNotes/1.3.2/BestNotes.BestNotes.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: BestNotes.BestNotes
+PackageVersion: 1.3.2
+PackageLocale: en-US
+Publisher: BestNotes
+PackageName: BestNotes
+License: BestNotes EULA
```
Comment (mdanish-kh) [inline manifests/b/BestNotes/BestNotes/1.3.2/BestNotes.BestNotes.locale.en-US.yaml:9]:
```suggestion
License: Proprietary
LicenseUrl: https://www.bestnotes.com/bestnotes-end-user-agreement/
```

File: manifests/b/BestNotes/BestNotes/1.3.2/BestNotes.BestNotes.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: BestNotes.BestNotes
+PackageVersion: 1.3.2
+PackageLocale: en-US
+Publisher: BestNotes
+PackageName: BestNotes
+License: BestNotes EULA
+Copyright: Copyright © 2019 BestNotes, LLC All rights reserved.
+ShortDescription: BestNotes Setup
```
Comment (mdanish-kh) [inline manifests/b/BestNotes/BestNotes/1.3.2/BestNotes.BestNotes.locale.en-US.yaml:11]:
```suggestion
ShortDescription: A Behavioral Health EHR that works for you!
```
ShortDescription should be meaningful and descriptive. Got this from the package homepage.

File: manifests/b/BestNotes/BestNotes/1.3.2/BestNotes.BestNotes.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+ReleaseNotes: InitialVersion
```
Comment (mdanish-kh) [inline manifests/b/BestNotes/BestNotes/1.3.2/BestNotes.BestNotes.locale.en-US.yaml:12]:
```suggestion
```
Please add the official ReleaseNotes with ReleaseNotesUrl - if you cannot find them, best to omit this entry.

File: manifests/b/BestNotes/BestNotes/1.3.2/BestNotes.BestNotes.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+PurchaseUrl: https://crm.bestnotes.com/portal/bestnotes/v1
```
Comment (mdanish-kh) [inline manifests/b/BestNotes/BestNotes/1.3.2/BestNotes.BestNotes.locale.en-US.yaml:13]:
```suggestion
PurchaseUrl: https://www.bestnotes.com/pricing/
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**